RealmSwift/Decimal128.swift

```diff
@@ -83,10 +83,16 @@ public final class Decimal128: RLMDecimal128, Decodable {
             throw DecodingError.dataCorruptedError(in: container, debugDescription: "Cannot convert value to Decimal128")
         }
     }
-}
 
-extension Decimal128 {
+    /// The mininum value for Decimal128
+    public static var min: Self {
+        minimumDecimalNumber as! Self
```

Comment:
Seems like minimum and maximum decimal number should both be refined for swift here.